R/helpers.R

```diff
@@ -708,6 +728,32 @@ cells_body <- function(columns = TRUE,
   cells
 }
 
+#' @noRd
+#' @export
+cells_data <- function(columns = TRUE,
+                       rows = TRUE) {
+
+  warning("The `cells_data()` function is deprecated and will soon be removed\n",
+          " * Use the `cells_body()` function instead",
+          call. = FALSE)
+
+  # Capture expressions for the `columns` and `rows` arguments
```

Comment:
Same here, could it just be the warning and then `cells_data({{columns}}, {{rows}})`?

Reply:
I tried this with `!!` but not with the double curly brace operator. Let me see if that makes a difference.

Reply:
Whoa. With the `{{ }}` we get an infinite loop!

Reply:
I think we may just have to keep the exact code in both places. The good thing is that I don't expect `cells_body()` or `cell_row_groups()` to change in ~6 months time (when I might change the `warning()` to a `stop()`).